korman/properties/prop_world.py

```diff
@@ -158,6 +191,10 @@ def export(self, exporter):
     use_texture_page = BoolProperty(name="Use Textures Page",
                                     description="Exports all textures to a dedicated Textures page",
                                     default=True)
+    age_name = StringProperty(name="Age Name",
+                              description="Name of the Age to be used for data files",
+                              default="Korman Age",
```

Comment:
Might be a good idea to leave this default blank, and marked invalid (red).  I didn't notice this field had been added and it even took me a moment to find it in the UI since it was already filled in with a name.  I've been exporting my Age with the wrong name!